### ISSUE TITLE
[APP SUBMITTED]: Unknown exception while loading URL http://thetvdb.com/api/F9C450E78D99172E/series/95011/all/en.zip: Traceback (most recent call last):
  File "/home/mediaman/.sickrage/lib/tvdb_api/tvdb_api.py", line 577, in _loadUrl
    resp = session.get(url.strip(), params=params)
  File "/home/mediaman/.sickrage/lib/requests/sessions.py", line 476, in get
    return self.request('GET', url, **kwargs)
  File "/home/mediaman/.sickrage/lib/requests/sessions.py", line 464, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/mediaman/.sickrage/lib/requests/sessions.py", line 576, in send
    r = adapter.send(request, **kwargs)
  File "/home/mediaman/.sickrage/lib/cachecontrol/adapter.py", line 36, in send
    cached_response = self.controller.cached_request(request)
  File "/home/mediaman/.sickrage/lib/cachecontrol/controller.py", line 102, in cached_request
    resp = self.serializer.loads(request, self.cache.get(cache_url))
  File "/home/mediaman/.sickrage/lib/cachecontrol/serialize.py

### ISSUE THREAD

**Issue by fishtenors** [SiCKRAGETV/sickrage-issues#3178]
### INFO
Python Version: **2.7.6 (default, Jun 22 2015, 17:58:13) [GCC 4.8.2]**
Operating System: **Linux-3.13.0-45-generic-x86_64-with-Ubuntu-14.04-trusty**
Locale: UTF-8
Branch: **master**
Commit: SiCKRAGETV/SickRage@638270689679f3958d1b593c329e7c6e5cb7448e
Link to Log: https://gist.github.com/fc62fdd71e665937f67b
### ERROR
```
SHOWQUEUE-ADD :: Show name with ID 95011 doesn't exist on theTVDB anymore. If you are using trakt, it will be removed from your TRAKT watchlist. If you are adding manually, try removing the nfo and adding again
```
---
_STAFF NOTIFIED_: @SiCKRAGETV/owners @SiCKRAGETV/moderators

**Comment by SiCKRAGEB0T** [comment]
## Thanks for the bug/issue report
Please make sure to submit all of the [required information](https://github.com/SiCKRAGETV/sickrage-issues/blob/master/README.md), or we won't be able to help you!
Provide as **much** information as you can such as debug logs, commit hash, and tell us how to replicate the problem.For basic **questions** use the [ [forum] ](https://sickrage.tv/forums/), and for **Feature Requests** [ [Feathub] ](http://feathub.com/SiCKRAGETV/SickRage)
Also make sure the title accurately describes your issue. ("SR not working" etc. is not ok!)

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [x] Post debug logs, either inline (for smaller logs) or using gist)

## Many issues can simply be solved by:
- Making sure you update to the latest version.
- Have you tried turning your device off and on again?
- Using the **search** function to see if this issue has already been reported/solved.
- Do a [GIT reset](https://github.com/SiCKRAGETV/sickrage-issues/wiki/FAQ%27s-and-Fixes#update-problems-try-this).
- Checking the [ [Wiki] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki) for [ [Guides] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Installation-&-Configuration-Guides), [ [FAQS] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki/FAQ%27s-and-Fixes) [ [Main settings] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Settings-explained), [ [Show settings] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Show-settings-explained) [ [Remaining settings] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Remaining-settings-explained)
- Shutting down SickRage and removing the `cache/mako` & `cache/sessions` folders.
- If SickRage doesnt find any episodes you probably need to :
 - add a [Scene Exception](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Scene-exceptions-and-numbering)
 - Set the [Default Episode Status](https://github.com/SiCKRAGETV/sickrage-issues/wiki/FAQ%27s-and-Fixes#newly-aired-shows-are-not-downloading-and-set-to-skippedignored) to `WANTED`
 - Make sure you understand how [quality detection](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Quality-Settings#quality-detectiondetermination) works during searches.
 - Shutdown SickRage and remove the `cache.db` file.
- If you use a QNAP and have SSL errors try this [package](http://apps.qnap.community/11-community/17-qsickrage).
- If you have a `'ascii' codec can't encode character` error, than set your Locale to [UTF-8](https://github.com/SiCKRAGETV/sickrage-issues/wiki/FAQ%27s-and-Fixes#i-have-problems-with-special-characters-%C3%A9-etc-what-can-i-do)

*ps. If you enjoy Sickrage and like the support provided, than please consider making a [donation](https://github.com/SiCKRAGETV/SickRage/wiki/Donations) to keep SickRage alive.*

**Comment by fishtenors** [comment]
So far, I have stopped SickRage and removed cache.db. I then stopped SickRage and cleared the mako & sessions folders. I also did a Git Reset. I received the error when adding the show after each of the steps.

A little background on the issue: The show in question is Modern Family. I kept seeing post-processing errors in the log for S07E01, so I deleted the show from the database, and checked the box to also delete files. I get the aforementioned error when trying to re-add the show. I have successfully added a different show, so it appears to be only affecting this show.

Let me know if I can provide anything else. Thanks!
